<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssignDownloadWidget</class>
 <widget class="QWidget" name="AssignDownloadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>分配下载</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="_firstStepLabel">
       <property name="text">
        <string>1.选择您想要测试分配</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="_testDataTableView">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="_secondStepLabel">
       <property name="text">
        <string>2.建议主题</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_filterLineEdit"/>
     </item>
     <item>
      <widget class="QListView" name="_subjListView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="_thirdStepLabel">
         <property name="text">
          <string>3.确认</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_assignTestPushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>分配测试</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
